packages/core/core-flows/src/order/workflows/return/confirm-return-request.ts

```diff
@@ -106,10 +106,6 @@ function prepareFulfillmentData({
 }
 
 function extractReturnShippingOptionId({ orderPreview, orderReturn }) {
-  if (!orderPreview.shipping_methods?.length) {
```

Comment:
We can't rely on this check, as the preview also contains the regular shipping methods of the order.

Reply:
Yeah, I can add that in 👍